project_national/national_upgrades.yml

```diff
@@ -112,6 +112,8 @@ upgrades:
       - option: Insulation Ceiling|R-60
         apply_logic:
           or:
+            - Insulation Ceiling|R-30
+            - Insulation Ceiling|R-19
```

Comment:
I currently have these upgrades sync'd between the national and testing upgrade projects. Should we extend the testing upgrades project as well?

Reply:
Sure. I'll do that.